docs/more/firewall.mdx

```diff
@@ -0,0 +1,61 @@
+# Firewall-Einstellungen für Sharezone
+
+Manchmal kann es der Fall sein, dass ein Netzwerk den Zugriff auf das Internet
+einschränkt (z.B. Schulen für das Schul-WLAN). Um Sharezone korrekt zu nutzen,
+müssen folgende Domains und Ports in der Firewall freigeschaltet werden.
+
```

Comment:
Maybe we could add after that, that the list is updated on an best effort basis and might not always be completely up-to-date.  
What do you think?

Reply:
I have moved the bottom text to the top and wrapped it around an info box, see https://docs.page/sharezoneapp/sharezone-app~667/more/firewall

Reply:
Best-effort doesn't mean "großmöglichster Mühe", more like "nach bestem Bemühen". Basically we try to keep it up-to-date if we can but it might fall under the table if we there is more important stuff that keeps us busy.

Reply:
I changed it ✅